src/firefly/html/demo/plotly-concepts.html

```diff
@@ -9,7 +9,7 @@
     <link rel=”apple-touch-startup-image” href=”images/fftools-ipad_splash_768x1004.png”>
     <title>Plotly Concept</title>
 
-    <script  type="text/javascript" src="../plotly-1.25.0.min.js"></script>
+    <script  type="text/javascript" src="../plotly-1.27.1.min.js"></script>
```

Comment:
Can't we not having an alias and get it from somewhere else that we maintain separately? github or directly from its CDN?